### PR TITLE
Update to work with bokeh 3.0

### PR DESCRIPTION
**Pull request by mattpap**
There are more issues to resolve, though they may be bokeh 3.0 unrelated.

### CODE REVIEW

**Comment by mattpap** [comment]
I guess that we have to merge this to enable the new actions' workflow.

**Comment by ndmlny-qs** [comment]
@mattpap you are correct. We could break this PR into separate components, one with the actions part, and the other where we continue to update for Bokeh 3.0. But, this sounds like unnecessary work. I'm doing a review now of what is currently in the PR. I think what you can have can be merged while we continue to work on the Bokeh 3.0 upgrade.

File: ipywidgets_bokeh/src/webpack.ts [hunk 1]
```diff
@@ -10,3 +10,5 @@ if (script instanceof HTMLScriptElement) {
     __webpack_require__.p = url
   }
 }
+
+export {}
```
Comment (ndmlny-qs) [inline ipywidgets_bokeh/src/webpack.ts:14]: I'm assuming this is required because of the new `isolatedModules` flag in `tsconfig.json`.

**Review by ndmlny-qs — APPROVED** [review]
lgtm

**Comment by mattpap** [comment]
I will merge this and see if the workflow works. I left `lint` and `tests` stubs in `package.json` to be filled-in in the future. I will copy lint setup from bokehjs. As to tests I don't have anything in mind yet.

**Pull request merged**